dashboard/app/views/layouts/application.html.haml

```diff
@@ -56,8 +56,8 @@
     - else
       - unless Rails.env.production?
         = render partial: 'layouts/environment_tag'
-    - if current_user && current_user.age.nil?
-      = render partial: 'layouts/age_interstitial'
+    - if current_user&.student? && (current_user.age.nil? || current_user.us_state.nil?)
```

Comment:
According to the comment in the Jira ticket, the new data collection is supposed to be for LTI students only. You can use `Policies::Lti.lti?` to determine if a user is an LTI user. Since we probably still want to collect age for non-LTI users, you might have to add some conditional logic to the popup itself, and only show the fields for gender and state to LTI users.